Cargo.toml

```diff
@@ -24,7 +24,7 @@ prost = "0.6"
 # Not available on WASM
 [target.'cfg(not(target_os = "unknown"))'.dependencies]
 miniupnpc = { version = "0.1", path = "miniupnpc" }
-
+base64 = "0.13"
```

Comment:
Move to the upper dependency block as this is for dependencies that can't be compiled to WebAssembly.